Sources/SwiftDependencyGraphs/DependencyGraph.swift

```diff
@@ -87,11 +89,12 @@ public struct DependencyGraph<V> where V: Hashable, V: Identifiable {
     }
   }
 
-  /// A neighbour of a vertex in a graph is a vertex that is connected to it by an edge.
+  /// A neighbour of a vertex is connected to it by an edge. For `.forwards` direction, it returns
+  /// only the outgoing edges. For `.backwards` direction, it returns only the incoming edges.
   /// - Parameters:
-  ///   - vertex: The vertex
-  ///   - direction: The direction of the edge
-  /// - Returns: The neighbours of the vertex
+  ///   - vertex: The vertex for which the neighbours are computed.
```

Comment:
Focusing on the effect/result more than on the implementation:

```suggestion
  ///   - vertex: The vertex of which all neighbours are returned.
```

Reply:
Done.